### PR TITLE
DOCS-12272-adding components and versions to table

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -146,55 +146,106 @@ Refer to xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypo
...
+
+* API Metadata Service v0.7.x
+* Asset Manager Service v0.8.x
+* Asset Portal Service v0.6.x
+* Assets Reviews Service v0.6.x
+* Custom Assets Facade Service v0.8.x
+* Customization Service v0.5.x
+* Dependency Resolver Service v0.6.x
+* File Service -
+* File Upload Service -
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/index.adoc]: Why do these two end with just a hyphen?

**Review by wwarner-salesforce — DISMISSED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/index.adoc]: Oh good catch, I need to remove those

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -146,55 +146,106 @@ Refer to xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypo
...
+a| 
+
+* API Metadata Service v0.7.x
+* Asset Manager Service v0.8.x
+* Asset Portal Service v0.6.x
+* Assets Reviews Service v0.6.x
+* Custom Assets Facade Service v0.8.x
+* Customization Service v0.5.x
+* Dependency Resolver Service v0.6.x
+* File Service -
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/index.adoc]:
```suggestion

```

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/index.adoc]:
```suggestion
```

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**